index.js

```diff
@@ -17,21 +17,14 @@ function errorCB (err) {
   if (err) throw err
 }
 
-module.exports = function (_db, opts, keys, path) {
+module.exports = function (_, opts, keys, path) {
+  //_ was legacy db. removed that, but for backwards compatibilty reasons do not change interface
   path = path || _db.location
```

Comment:
How about also removing or changing this reference to `_db`?

Reply:
fixed.